support/android/apk/project.properties

```diff
@@ -1 +0,0 @@
-target=android-18
```

Comment:
I'm going a little ballsy here, and assuming that the contents of `support/android/apk` are built with the variable specified [here](https://github.com/servo/servo/pull/11278/files#diff-40f5a7cf22f94aad059b2c1795347f5eR104).

Notice that in [`build.xml`](https://github.com/servo/servo/pull/11278/files#diff-2419f69b360f1fa393abc46d9d331d01R39), `project.properties` is no longer loaded, and `target` is assumed from the environment variable `$ANDROID_TARGET`